feat: Add vsphere-supervisor builder (#220)
hashicorp/packer-plugin-vsphere: docs/builders/index.mdx

The vSphere Packer builder is able to create vSphere virtual machines for use
with any VMware product.

> Packer actually comes with three builders able to create vSphere machines,

Reviewer: ```suggestion
This Packer plugin includes three builders to create vSphere machines,
```
